app/views/static_pages/home.html.erb

```diff
@@ -1,92 +1,110 @@
-              <%= link_to " Join Group Problem Solving", user_viewer_path(@hostresearch.permalink, chatrm: @displayresearch.topic, event: @displayresearch.id ), class: "btn btn-primary btn-sm fa fa-video-camera", style: "margin-top: 10px; margin-bottom: 10px; font-size: 42px; border-color:#ffc627; border-radius: 5px;" %>
-            </div>
-          <% end %>
-        <% end %>
-        <% if @displaystudy.present? && @displayresearch.present? %>
-          <% if !(Time.current > Time.parse(@start_timeresearch)) && (Time.current < (Time.parse(@start_timeresearch) + 1.hour)) %>
-            <% if !(Time.current > Time.parse(@start_timestudy)) && (Time.current < (Time.parse(@start_timestudy) + 1.hour)) %>
-              <h2 id="buttonTitle">
-              <h2 id="buttonTitle" style="margin-bottom: 30px">
-              <%= link_to 'See an example Video Chat Room', "/codingwhilefemale/viewer?chatrm=Conversation", class: "btn btn-lg btn-primary", style: "border-radius: 5px;", title: "Check your webcam" %>
-              </h2>
-            <% end %>
-          <% end %>
-        <% end %>        <br><br>
+<div class="row mr-0 ml-0 pl-0 pr-0" style="background-color: #8C1D40">
+  <div class="alert" style="background-color: #8C1D40"></div>
+  <div class="col-sm">
+    <% flash.each do |key, value| %>
+      <div class="<%= flash_class(key) %>">
+        <%= value %>
       </div>
+    <% end %>
+    <h1 class="white">
+      Discover Activism that makes you
+      <div>
+        <%= image_tag("GoldThinQ60.png", style: "padding-left: 150px;")%><br> </div>
+      </h1>
+      <h2 class="new" style="color:white">
+        Crowdsource Activism via Video Chat
+      </h2>
     </div>
-
-<div class="col-lg-7 col-md-12" id="countdown_clock">
-  <div class="main row">
-    <br>
-    <div class="col-lg-3 col-md-6">
-      <h2>Conversation Schedule</h2>
-      <h4>Discover topics that motivate you!</h4>
-      <% if user_signed_in? %>
-        <%= link_to 'Post your Own Conversation', new_event_path, class: "btn btn-warning", title: "Host drop-in conversation sessions or invite people to ask questions about your area of expertise" %>
-      <% end %>
-    </div>
-    <div class="col-lg-3 col-md-6">
-      <%= render 'layouts/countdown_clock' %>
+    <div class="col-sm">
+      <%= image_tag("header-image.png", style: "width:80%; padding-top:10px; padding-bottom:10px;")%>
     </div>
+
   </div>
-  <div class="tabContainer">
-    <div class="buttonContainer">
-      <button onclick="showPanel(0,'#4caf50')">List View</button>
-      <button onclick="showPanel(1,'#4caf50')">Daily View</button>
-      <button onclick="showPanel(2,'#4caf50')">Monthly View</button>
+  <!-- end of top section-->
+
+  <div class="col-sm">
+    <h1 class="new">
+      New to ThinQ.tv?
+    </h1>
+    <h3 class="Subtitle">
+      Most of the day we have a video chat
+      <%= link_to "Activism Hall", studyhall_path, style: "color: #ffc627" %>
```

Comment:
I think this should not be "Activism Hall" and Lisa advised that the name of the buttons/links should not be hard-coded.

Reply:
We certainly can't have hardcoded links. However, I looked at home.html.erb and did not see hardcoded links